CPT331.Data/CrimeRepository.cs

```diff
@@ -139,6 +171,32 @@ public List<Crime> GetCrimes()
 
 			return crimes;
 		}
+		
+		/// <summary>
+		/// Exports a list of ReadOnlyDataObject types to a stream.
+		/// </summary>
+		/// <param name="readOnlyDataObjects">The list of ReadOnlyDataObject objects to export.</param>
+		/// <param name="stream">The stream to export the ReadOnlyDataObject list to.</param>
+		protected override void OnExport(List<ReadOnlyDataObject> readOnlyDataObjects, Stream stream)
```

Comment:
So this doesn't work quite the way I want it to yet - ideally it should stream from the database to the browser - which it doesn't. Hopefully can be tweaked in a later PR.